src/shared_utilities/drivers.jl

```diff
@@ -259,16 +259,18 @@ function surface_fluxes_at_a_point(
     cp_d::FT = Thermodynamics.Parameters.cp_d(thermo_params)
     R_d::FT = Thermodynamics.Parameters.R_d(thermo_params)
     T_0::FT = LSMP.T_0(earth_param_set)
+    _LH_v0::FT = LSMP.LH_v0(earth_param_set)
     cp_m = Thermodynamics.cp_m(thermo_params, ts_in)
     T_in = Thermodynamics.air_temperature(thermo_params, ts_in)
     ΔT = T_in - T_sfc
     hd_sfc = cp_d * (T_sfc - T_0) + R_d * T_0
     E0 = SurfaceFluxes.evaporation(surface_flux_params, sc, conditions.Ch)
     r_ae = 1 / (conditions.Ch * SurfaceFluxes.windspeed(sc))
+    ρ_air = Thermodynamics.air_density(thermo_params, ts_in)
     E = E0 * r_ae / (r_sfc + r_ae)
     Ẽ = E / _ρ_liq
-    H = conditions.shf + hd_sfc * (E0 - E)
-    LH = conditions.lhf * r_ae / (r_sfc + r_ae)
+    H = -ρ_air * cp_m * ΔT / r_ae#conditions.shf + hd_sfc * E0
```

Comment:
@braghiere here are the expressions we use for LHF and SHF. just the standard ones without the extra terms clima SurfaceFluxes.jl adds